### PR TITLE
DOC: Stop ignoring fixed rst files in flake8-rst

### PR DESCRIPTION
**Pull request by datapythonista**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

Removing from the list of ignored files in flake8-rst, the pages that have been fixed before the ignore list was merged.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -69,25 +69,19 @@ exclude =
...
-    doc/source/comparison_with_stata.rst
-    doc/source/computation.rst
     doc/source/contributing.rst
     doc/source/contributing_docstring.rst
     doc/source/dsintro.rst
     doc/source/enhancingperf.rst
     doc/source/extending.rst
     doc/source/groupby.rst
     doc/source/indexing.rst
-    doc/source/io.rst
```
Comment (FHaase) [inline setup.cfg]: `io.rst` isn't yet completely fixed.

File: setup.cfg [hunk 2]
```diff
@@ -69,25 +69,19 @@ exclude =
...
     doc/source/merging.rst
     doc/source/missing_data.rst
     doc/source/options.rst
     doc/source/release.rst
     doc/source/reshaping.rst
     doc/source/timedeltas.rst
```
Comment (FHaase) [inline setup.cfg]: `timedeltas.rst` can also be removed

**Comment by jreback** [comment]
@datapythonista can you rebase this. (i merged something else)

**Comment by datapythonista** [comment]
@jreback this is green now

**Pull request merged**